tests/checkout/conflict.c

```diff
@@ -50,13 +50,13 @@ struct checkout_index_entry {
 	uint16_t mode;
 	char oid_str[GIT_OID_HEXSZ+1];
 	int stage;
-	char path[128];
+	char path[256];
```

Comment:
Why don't you just use `PATH_MAX` here and in the following declarations?

Reply:
I think these are not actually paths, but filenames.  So NAME_MAX + 1  is probably what's wanted.